sktime/forecasting/compose/_ensemble.py

```diff
@@ -2,13 +2,176 @@
 # -*- coding: utf-8 -*-
 # copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
 
-__author__ = ["Markus Löning"]
-__all__ = ["EnsembleForecaster"]
+__author__ = ["Markus Löning", "Martin Walter"]
```

Comment:
the author string should have your GitHub username, so it'll be "aiwalter"

Reply:
actually we should maybe replace them all everywhere in a separate PR